exporters/zipkin/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenTelemetry - Zipkin Span Exporter
+
+[![Javadocs][javadoc-image]][javadoc-url]
+
+This is the OpenTelemetry exporter, which sends span data to Zipkin.
+
+## Configuration
+
+The Zipkin span exporter can be configured programmatically.
+
+An example of simple Zipkin exporter initialization. In this case
+spans will be sent to a Zipkin endpoint running on `localhost`:
+
+```java
+    ZipkinExporterConfiguration configuration =
+        ZipkinExporterConfiguration.builder()
+            .setV2Url("https://localhost/api/v2/spans")
```

Comment:
probably a wildcard s/http/https/ here. Fixing to avoid first timers getting confused why things don't work.

```suggestion
            .setV2Url("http://localhost/api/v2/spans")
```